procrastinate/sql/migrations/02.00.03_01_add_job_priority.sql

```diff
@@ -0,0 +1,124 @@
+ALTER TABLE procrastinate_jobs ADD COLUMN priority integer DEFAULT 0 NOT NULL;
+
+CREATE UNIQUE INDEX idx_procrastinate_jobs_priority_id ON procrastinate_jobs (priority, id) WHERE status = 'todo';
```

Comment:
You removed the index from the schema.sql file but not from the migration.

Reply:
Uups 🙄. Done.